src/main/resources/datical/datical_deploy.bat.ftl

```diff
@@ -10,4 +10,18 @@
 
 -->
 <#include "/datical/datical_generic.ftl">
+<#include "/datical/datical_credentials.bat.ftl">
+${ddb_audit_user}
```

Comment:
Are these 4 lines needed? Shouldn't the include be enough? So taking out the assign from the include statement?